src/shared/utils/local-data-storage.ts

```diff
@@ -0,0 +1,24 @@
+export class LocalDataStorage<DataType> {
+  public localStorageKeyPrefix: string = "";
+
+  constructor(localStorageKeyPrefix: string) {
+    this.localStorageKeyPrefix = localStorageKeyPrefix;
+  }
+
+  public load(uuid = ""): DataType | undefined {
+    let data: DataType | undefined;
+    const stringifiedData = window.localStorage.getItem(this.localStorageKeyPrefix + uuid);
+    if (stringifiedData) {
+      try {
+        data = JSON.parse(stringifiedData);
+        // tslint:disable-next-line:no-empty
+      } catch (e) {
```

Comment:
If you add a comment in the catch clause, e.g.
```
} catch (e) {
  // ignore errors
}
```
I don't think you'd need the `// tslint:disable-next-line:no-empty`